### PR TITLE
Change url_validator gem url [Fixes #114080323]

### PR DESCRIPTION
**Pull request by Maroo-b**
@tansaku I edited Gemfile to use AgileVentures url_validator fork and made the required changes

### CODE REVIEW

**Comment by tansaku** [comment]
@Maroo-b this looks great, but I notice we seem to be upgrading to the latest Rails at the same time.  Naturally that's great, but it would make sense to avoid wrapping those two things up in the same commit ...?

File: app/models/base_organisation.rb [hunk 1]
```diff
@@ -3,9 +3,9 @@ class BaseOrganisation < ActiveRecord::Base
   friendly_id :slug_candidates, use: :slugged
   
   acts_as_paranoid
-  validates_url :website, prefferred_scheme: 'http://', message: 'Website is not a valid URL',
+  validates_url :website, preferred_scheme: 'http://', message: 'Website is not a valid URL',
     if: proc{|org| org.website.present?}
-  validates_url :donation_info, prefferred_scheme: 'http://', message: 'Donation url is not a valid URL',
+  validates_url :donation_info, preferred_scheme: 'http://', message: 'Donation url is not a valid URL',
```
Comment (tansaku) [inline app/models/base_organisation.rb]: could we just break this over two lines - putting the message section on the next line?

**Comment by Maroo-b** [comment]
@tansaku I'll check why the update for Rails happened because I updated the `url_validator` gem only

**Comment by tansaku** [comment]
@Maroo-b I managed to get a review app deploy for this PR but I had to kick it off manually - have turned those settings off and on again to see if that helps ...

**Comment by Maroo-b** [comment]
@tansaku interesting maybe we should verify Heorku documentation to check how to kick it off on PR

**Comment by tansaku** [comment]
@Maroo-b I turned it on and off again:

**Comment by tansaku** [comment]
ah, hmm from https://devcenter.heroku.com/articles/github-integration-review-apps

```Alternatively, you can enable automatic review app creation. That will cause Heroku to create and deploy apps as soon as pull requests are opened on GitHub. For security and billing reasons, Heroku will not automatically create review apps for pull requests to public repos that are sent from forks. Review apps can still be created manually for these pull requests.```

**Comment by tansaku** [comment]
I have this setting also:



hmm so maybe it will only make review apps for PRs if they are coming from the same repo

**Comment by Maroo-b** [comment]
@tansaku, that's great so for the moment if we need a review app to share with client we can create on manually ( maybe some functionalities will not work like sending emails )

**Comment by tansaku** [comment]
so @Maroo-b did you want to extract the rails upgrade from this?  or shall I just go ahead and merge?  I guess we want to be on the latest rails anyway ...

**Comment by Maroo-b** [comment]
@tansaku I tried to update the gem in a new branch but it seems that rails is updated too (strange), maybe we should merge it and we gain some additional fixes for Rails :)

**Comment by tansaku** [comment]
yeah :-)

**Pull request merged**